templates/partials/question.html

```diff
@@ -58,12 +59,19 @@
 {% set question_guidance %}
   {%- if question.guidance -%}
     {% set contents = question.guidance.contents %}
-    {% call onsPanel({
-      "id": "question-guidance-" ~ question.id,
-      "classes": "ons-u-mb-m"
-    }) %}
-      {% include 'partials/contents.html' %}
-    {% endcall %}
+    {% for item in contents %}
+      {%- if item["title"] or item["description"] or item["list"] -%}
```

Comment:
Non blocker but I've just checked and contents.html also supports "definition" field and it is accepted as a field in contents in validator but can it be used here? Should we check for item["definition"]? Does guidance support "definition"?

Reply:
Updated the if statement as discussed 👍